README.md

```diff
@@ -55,6 +55,33 @@ In ysplitter mode a third virtual port is connected and has access to the data s
 
 ![PICsimLab](ssniffer/tty0ttySerialSniffer.png "PICsimLab")  
 
+## Quickstart guide
```

Comment:
I think it would be better to change the title to something like "precompiled .deb packages".

Reply:
Yes of course